apps/cow-amm-deployer/src/components/TokenLogo.tsx

```diff
@@ -36,11 +50,27 @@ export const TokenLogo = ({
   className,
   quality,
 }: ImageFallbackProps) => {
-  const [imageSrc, setImageSrc] = useState<string>();
+  const [imagesSrc, setImagesSrc] = useState<string[]>([
```

Comment:
🤔 why do we need state here?

Reply:
we have a few options of image source for each token, this state should control it.